svgs/nekogram_x.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg">
+    <path style="fill:#000" d="M20.07 13.13a1.133.954 54.911 0 1-.13 1.476 1.133.954 54.911 0 1-1.432-.379 1.133.954 54.911 0 1 .13-1.475 1.133.954 54.911 0 1 1.431.379zm-5.278 4.138a1.133.954 54.911 0 1-.13 1.475 1.133.954 54.911 0 1-1.431-.378 1.133.954 54.911 0 1 .13-1.476 1.133.954 54.911 0 1 1.431.379zm-3.854 4.689a10.03 10.03 0 0 1-5.827-2.705C1.58 15.873.982 10.47 3.689 6.423 5.055 4.38 7.182 2.88 9.542 2.291c.9-.224 1.463-.291 2.462-.291 1.035 0 1.589.07 2.542.318 4.918 1.283 8.089 6.096 7.346 11.153a9.985 9.985 0 0 1-5.51 7.517 9.593 9.593 0 0 1-3.196.95c-.642.075-1.676.083-2.248.019zm1.953-.228c1.567-.153 3.157-.708 4.441-1.55a9.71 9.71 0 0 0 3.43-3.9c.695-1.428.981-2.683.98-4.292-.001-1.202-.148-2.095-.52-3.178-.296-.857-.391-1.03-.663-1.202-.95-.6-2.79-1.371-3.466-1.451-.304-.037-.363.033-.363.424.001.538.218 2.189.49 3.725.085.482.103.684.068.75-.026.049-.282.212-.57.363-1.09.574-1.466.831-2.252 1.539-.4.36-.449.39-.626.387-.303-.005-.637-.158-1.605-.733-1.295-.77-1.846-1.022-2.347-1.075-.312-.033-.46.008-.5.138-.07.239-.106 1.47-.069 2.43.066 1.712.103 1.908.665 3.487.34.956.347.986.487 2.014.136.997.254 1.498.455 1.927.073.155.105.178.29.202.35.047 1.169.045 1.675-.005z"/>
+</svg>
```

Comment:
```suggestion
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   width="24"
   height="24"
   viewBox="0 0 24 24"
   fill="none"
   version="1.1"
   id="svg4"
   sodipodi:docname="nekogram_x (copy).svg"
   inkscape:version="1.1.1 (3bf5ae0d25, 2021-09-20)"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <defs
     id="defs8" />
  <sodipodi:namedview
     id="namedview6"
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1.0"
     inkscape:pageshadow="2"
     inkscape:pageopacity="0.0"
     inkscape:pagecheckerboard="0"
     showgrid="false"
     inkscape:zoom="32"
     inkscape:cx="4.625"
     inkscape:cy="12.796875"
     inkscape:window-width="2560"
     inkscape:window-height="1015"
     inkscape:window-x="0"
     inkscape:window-y="32"
     inkscape:window-maximized="1"
     inkscape:current-layer="svg4" />
  <path
     id="circle4919"
     style="fill:#000000;stroke-width:0.210526;stroke-linecap:round;stroke-linejoin:round"
     d="M 12,2 A 10,10 0 0 0 2,12 10,10 0 0 0 12,22 10,10 0 0 0 22,12 10,10 0 0 0 12,2 Z m 8.42e-4,0.789474 a 9.2105263,9.2105263 0 0 1 7.541118,3.927221 l -0.01398,-0.0093 c 0.337642,0.48544 0.631157,0.955798 0.570107,0.955798 -0.03258,1e-6 -0.300258,-0.122137 -0.594779,-0.271381 C 19.208786,7.242566 18.684459,7.02983 18.338013,6.919155 17.110709,6.527078 17.032486,6.615727 17.192042,8.21747 c 0.117194,1.176486 0.354142,2.595524 0.489515,2.931331 0.0557,0.138163 -0.05274,0.240144 -0.488692,0.460321 -0.706288,0.35671 -1.354853,0.792264 -2.047081,1.374384 -0.298013,0.25061 -0.58836,0.455592 -0.645148,0.455592 -0.05679,0 -0.687344,-0.301136 -1.40111,-0.669203 -0.713763,-0.368068 -1.567119,-0.777406 -1.896382,-0.909744 -0.90528,-0.363852 -0.924137,-0.329595 -0.924137,1.685238 0,1.881616 0.154147,2.893441 0.627674,4.123149 0.172873,0.448931 0.383565,1.252367 0.468338,1.785362 0.08477,0.532995 0.198618,1.14627 0.253084,1.362871 l 0.0991,0.393708 -0.01563,-0.02076 A 9.2105263,9.2105263 0 0 1 2.790315,11.999957 9.2105263,9.2105263 0 0 1 12.000841,2.789431 Z m 7.469366,10.030427 c 0.301353,-0.0098 0.607873,0.145424 0.804483,0.416119 0.306782,0.422306 0.251187,0.985709 -0.124178,1.258429 -0.375362,0.272712 -0.928359,0.151491 -1.235197,-0.270765 -0.306951,-0.422298 -0.251443,-0.985843 0.123971,-1.258635 0.08927,-0.06485 0.192034,-0.10928 0.302427,-0.130756 0.0425,-0.0083 0.08544,-0.013 0.128494,-0.01439 z m -4.919407,3.869655 c 0.301293,-0.0097 0.607701,0.145495 0.804276,0.416119 0.306782,0.422306 0.251187,0.985709 -0.124178,1.258429 -0.375362,0.272712 -0.928359,0.151491 -1.235197,-0.270765 -0.306918,-0.422354 -0.251317,-0.985914 0.124178,-1.258635 0.08927,-0.06485 0.192033,-0.109279 0.302425,-0.130757 0.0425,-0.0083 0.08545,-0.013 0.128496,-0.01439 z" />
</svg>
```

Reply:
Here’s an updated suggestion, as per [this comment](https://github.com/LawnchairLauncher/lawnicons/pull/51#discussion_r748908481).

```suggestion
<vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:width="24dp"
    android:height="24dp"
    android:viewportWidth="24"
    android:viewportHeight="24">
  <path
      android:pathData="M12,2C6.48,2 2,6.48 2,12 2,17.52 6.48,22 12,22 17.52,22 22,17.52 22,12 22,6.48 17.52,2 12,2ZM20.6261,7.9889c0.5672,1.2188 0.8861,2.5773 0.8861,4.011 0,5.2563 -4.2559,9.5122 -9.5122,9.5122 -0.302,0 -0.6008,-0.0163 -0.8956,-0.0438 -1.441,-0.2285 -1.3499,-0.1789 0,0 -0.0704,-0.0246 -0.1042,-0.0671 -0.1524,-0.1696 -0.1961,-0.4185 -0.3113,-0.9061 -0.444,-1.8788 -0.1366,-1.0029 -0.1428,-1.0319 -0.4745,-1.9646 -0.5483,-1.5405 -0.5854,-1.733 -0.6498,-3.4032 -0.0361,-0.9366 0.0003,-2.1373 0.0686,-2.3704 0.039,-0.1268 0.1834,-0.1675 0.4878,-0.1353 0.4888,0.0517 1.0251,0.2987 2.2885,1.0499 0.9444,0.561 1.2707,0.7097 1.5663,0.7146 0.1727,0.0029 0.2214,-0.0261 0.6117,-0.3773 0.7668,-0.6907 1.1336,-0.9415 2.197,-1.5015 0.281,-0.1473 0.5291,-0.3066 0.5545,-0.3544 0.0342,-0.0644 0.0181,-0.2615 -0.0648,-0.7317 -0.2654,-1.4985 -0.4773,-3.1089 -0.4783,-3.6338 0,-0.2861 0.032,-0.3955 0.1734,-0.4173l0,-0c0.0471,-0.0073 0.1069,-0.0052 0.181,0.0038 0.6595,0.078 2.4535,0.8304 3.3803,1.4158 0.1143,0.0723 0.1966,0.1484 0.2763,0.2744m-1.6654,4.5998c-0.0485,-0.0005 -0.097,0.0021 -0.1448,0.0096 -0.1241,0.0194 -0.2393,0.0653 -0.3392,0.1353 -0.4207,0.295 -0.4778,0.9389 -0.1277,1.4386 0.3508,0.4994 0.9761,0.6649 1.3967,0.3697 0.4207,-0.295 0.4778,-0.9389 0.1277,-1.4386h-0.002c-0.2246,-0.3195 -0.5714,-0.5111 -0.9108,-0.5145zM13.8121,16.6227c-0.0486,-0.0004 -0.097,0.0039 -0.1448,0.0114 -0.1241,0.0194 -0.2393,0.0653 -0.3392,0.1353 -0.4219,0.2949 -0.4791,0.9403 -0.1277,1.4405 0.3514,0.4989 0.9766,0.6635 1.3967,0.3678 0.4207,-0.295 0.4778,-0.9389 0.1277,-1.4386 -0.225,-0.3201 -0.5728,-0.5136 -0.9127,-0.5164z"
      android:strokeWidth="0.97561"
      android:fillColor="#000000"/>
</vector>
```